go.mod

```diff
@@ -1,9 +1,12 @@
-module github.com/VoltDB/voltdb-client-go
+module voltdb-client-go
 
 go 1.16
 
+replace github.com/VoltDB/voltdb-client-go => ../voltdb-client-go
```

Comment:
remove replace this is for your local testing

Reply:
Do you need to modify go mod and sum? are you bringing new packages?

Reply:
Reverting the changes to go.mod and go.sum